lighthouse-core/test/audits/uses-rel-preload-test.js

```diff
@@ -209,6 +209,18 @@ describe('Performance: uses-rel-preload audit', () => {
       assert.equal(output.details.items.length, 0);
     });
   });
+  
```

Comment:
```suggestion

```

Reply:
Travis is mad about this trailing space.  I think this suggestion is accurate to just remove the space but keep the /n